src/main/scala/com/monsanto/arch/cloudformation/model/resource/SSM.scala

```diff
@@ -173,6 +173,8 @@ object `AWS::SSM::Document` extends DefaultJsonProtocol {
                     mainSteps:   Option[Seq[DocumentStep]],
                     description: Option[Token[String]] = None,
                     parameters:  Option[Map[String, DocumentParameter]] = None,
+                    outputs:     Option[Seq[Token[String]]],
```

Comment:
Should this default to None for backwards compatibility?

Reply:
Yes, it should.  It has been changed.